packages/features/bookings/components/AvailableTimes.tsx

```diff
@@ -110,15 +114,22 @@ export const AvailableTimes = ({
               {dayjs.utc(slot.time).tz(timezone).format(timeFormat)}
               {bookingFull && <p className="text-sm">{t("booking_full")}</p>}
               {hasTimeSlots && !bookingFull && (
-                <p className="flex items-center text-sm lowercase">
+                <p className={`flex items-center text-sm ${showAvailableSeatsCount && "lowercase"}`}>
                   <span
                     className={classNames(colorClass, "mr-1 inline-block h-2 w-2 rounded-full")}
                     aria-hidden
                   />
-                  {slot.attendees ? seatsPerTimeSlot - slot.attendees : seatsPerTimeSlot}{" "}
-                  {t("seats_available", {
-                    count: slot.attendees ? seatsPerTimeSlot - slot.attendees : seatsPerTimeSlot,
-                  })}
+                  {showAvailableSeatsCount
+                    ? `${availableSeatsCount} ${t("seats_available", {
+                        count: availableSeatsCount,
+                      })}`
+                    : isNearlyFull
+                    ? t("seats_nearly_full")
+                    : isHalfFull
+                    ? t("seats_half_full")
+                    : t("seats_available", {
+                        count: availableSeatsCount,
+                      })}
```

Comment:
Do you think we could make this into a component of some sort? Seems to be repeated across the app in a few places

Reply:
That's a good idea! I've refactored the code to create a component out of it.